src/chrome/content/rules/Geekzone.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+        Served over HTTP:
+			feeds.geekzone.co.nz
+			m.geekzone.co.nz
+			price.geekzone.co.nz
+			www.geekzone.co.nz
+
+		Available over HTTPS (301 redirect from HTTP)
+			cdn.geekzone.co.nz
+			images.geekzone.co.nz
+			live.geekzone.co.nz
+			testip.geekzone.co.nz
+
+		Available over HTTPS only, no need for rewrite

```

Comment:
Thanks for your comment J0WI. I don't quite understand the context - is it related to the "Available over HTTPS only" line or the ones before?